### PR TITLE
chore(fluxdocs): add docs for internal and misc pacakges

### PR DESCRIPTION
**Pull request by nathanielc**
There were several packages we had not yet documented because they were
for testing or internal use only. This PR adds docs to those packages so
that they pass the linter checks.

### Done checklist
- [ ] docs/SPEC.md updated
- [ ] Test cases written

### CODE REVIEW

File: stdlib/internal/debug/debug.flux [hunk 1]
```diff
@@ -1,26 +1,54 @@
...
+//
+// ## Parameters
+// - tables: Stream to consume into single buffers per table.
+//
 builtin slurp : (<-tables: stream[A]) => stream[A] where A: Record
 
 // sink will discard all data that comes into it.
+//
+// ## Parameters
+// - tables: Stream to discard
```
Comment (sanderson) [inline stdlib/internal/debug/debug.flux:34]:
```suggestion
// - tables: Stream to discard.
```

File: stdlib/internal/debug/debug.flux [hunk 2]
```diff
@@ -1,26 +1,54 @@
...
+//
 builtin sink : (<-tables: stream[A]) => stream[A] where A: Record
 
+// getOption gets the value of an option using a form of reflection
+//
+// ## Parameters
+// - pkg: Full path of the package
+// - name: Name of the option
```
Comment (sanderson) [inline stdlib/internal/debug/debug.flux:42]:
```suggestion
// getOption gets the value of an option using a form of reflection.
//
// ## Parameters
// - pkg: Full path of the package.
// - name: Option name.
```

File: stdlib/internal/debug/debug.flux [hunk 3]
```diff
@@ -1,26 +1,54 @@
...
+//
 builtin getOption : (pkg: string, name: string) => A
 
 // feature returns the value associated with the given feature flag.
+//
+// ## Parameters
+// - key: Name of the feature flag
```
Comment (sanderson) [inline stdlib/internal/debug/debug.flux:49]:
```suggestion
// - key: Feature flag name.
```

File: stdlib/internal/gen/gen.flux [hunk 4]
```diff
@@ -1,6 +1,13 @@
+// Package gen provides methods for generating data.
 package gen
 
 
+// tables generates a stream of table data
```
Comment (sanderson) [inline stdlib/internal/gen/gen.flux:5]:
```suggestion
// tables generates a stream of table data.
```

File: stdlib/internal/gen/gen.flux [hunk 5]
```diff
@@ -1,6 +1,13 @@
...
+//
+// ## Parameters
+// - n: Number of rows to generate.
+// - nulls: Percentage chance that a null value will be used in the input. This should be a number between 0 and 1.
+// - tags: Set of tags with their cardinality that should be generated.
```
Comment (sanderson) [inline stdlib/internal/gen/gen.flux:10]:
```suggestion
// - nulls: Percentage chance that a null value will be used in the input. Valid value range is `[0.0 - 1.0]`.
// - tags: Set of tags with their cardinality to generate.
```

File: stdlib/internal/influxql/influxql.flux [hunk 6]
```diff
@@ -1,6 +1,12 @@
+// Package influxql provides some constants
```
Comment (sanderson) [inline stdlib/internal/influxql/influxql.flux:1]:
```suggestion
// Package influxql provides constants for working with InfluxQL.
```

File: stdlib/internal/influxql/influxql.flux [hunk 7]
```diff
@@ -1,6 +1,12 @@
...
 package influxql
 
 
+// epoch is the absolute time that all InfluxQL time and duration values use as a zero reference.
 epoch = 1970-01-01T00:00:00Z
+
+// minTime is the earliest time InfluxQL can represent
 minTime = 1677-09-21T00:12:43.145224194Z
+
+// maxTime is the latest time InfluxQL can represent
```
Comment (sanderson) [inline stdlib/internal/influxql/influxql.flux:11]:
```suggestion
// minTime is the earliest time InfluxQL can represent.
minTime = 1677-09-21T00:12:43.145224194Z

// maxTime is the latest time InfluxQL can represent.
```

File: stdlib/internal/promql/promql.flux [hunk 8]
```diff
@@ -6,65 +8,94 @@ import "math"
 import "universe"
 import "experimental"
 
-// changes() implements functionality equivalent to PromQL's changes() function:
+// changes implements functionality equivalent to PromQL's changes() function:
 //
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#changes
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:13]:
```suggestion
// changes implements functionality equivalent to
// [PromQL's `changes()` function](https://prometheus.io/docs/prometheus/latest/querying/functions/#changes).
```

File: stdlib/internal/promql/promql.flux [hunk 9]
```diff
@@ -6,65 +8,94 @@ import "math"
...
+//
+// ## Parameters
+// - tables: Stream of tables.
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:16]:
```suggestion
// - tables: Input data. Default is piped-forward data (`<-`).
```

File: stdlib/internal/promql/promql.flux [hunk 10]
```diff
@@ -6,65 +8,94 @@ import "math"
...
 builtin changes : (<-tables: stream[{A with _value: float}]) => stream[{B with _value: float}]
 
-// promqlDayOfMonth() implements functionality equivalent to PromQL's day_of_month() function:
+// promqlDayOfMonth implements functionality equivalent to PromQL's day_of_month() function:
 //
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#day_of_month
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:21]:
```suggestion
// promqlDayOfMonth implements functionality equivalent to 
// [PromQL's `day_of_month()` function](https://prometheus.io/docs/prometheus/latest/querying/functions/#day_of_month).
```

File: stdlib/internal/promql/promql.flux [hunk 11]
```diff
@@ -6,65 +8,94 @@ import "math"
...
+//
+// ## Parameters
+// - timestamp: Time as a floating point value.
 builtin promqlDayOfMonth : (timestamp: float) => float
 
-// promqlDayOfWeek() implements functionality equivalent to PromQL's day_of_week() function:
+// promqlDayOfWeek implements functionality equivalent to PromQL's day_of_week() function:
 //
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#day_of_week
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:29]:
```suggestion
// promqlDayOfWeek implements functionality equivalent to
// [PromQL's `day_of_week()` function](https://prometheus.io/docs/prometheus/latest/querying/functions/#day_of_week).
```

File: stdlib/internal/promql/promql.flux [hunk 12]
```diff
@@ -6,65 +8,94 @@ import "math"
...
+//
+// ## Parameters
+// - timestamp: Time as a floating point value.
 builtin promqlDayOfWeek : (timestamp: float) => float
 
-// promqlDaysInMonth() implements functionality equivalent to PromQL's days_in_month() function:
+// promqlDaysInMonth implements functionality equivalent to PromQL's days_in_month() function:
 //
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#days_in_month
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:37]:
```suggestion
// promqlDaysInMonth implements functionality equivalent to
// [PromQL's `days_in_month()` function](https://prometheus.io/docs/prometheus/latest/querying/functions/#days_in_month).
```

File: stdlib/internal/promql/promql.flux [hunk 13]
```diff
@@ -6,65 +8,94 @@ import "math"
...
+// ## Parameters
+// - timestamp: Time as a floating point value.
 builtin promqlDaysInMonth : (timestamp: float) => float
 
-// emptyTable() returns an empty table, which is used as a helper function to implement
+// emptyTable returns an empty table, which is used as a helper function to implement
 // PromQL's time() and vector() functions:
 //
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#time
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#vector
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:47]:
```suggestion
// emptyTable returns an empty table, which is used as a helper function to implement
// PromQL's [`time()`](https://prometheus.io/docs/prometheus/latest/querying/functions/#time) and
// [`vector()`](https://prometheus.io/docs/prometheus/latest/querying/functions/#vector) functions.
```

File: stdlib/internal/promql/promql.flux [hunk 14]
```diff
@@ -6,65 +8,94 @@ import "math"
...
 builtin emptyTable : () => stream[{_start: time, _stop: time, _time: time, _value: float}]
 
-// extrapolatedRate() is a helper function that calculates extrapolated rates over
+// extrapolatedRate is a helper function that calculates extrapolated rates over
 // counters and is used to implement PromQL's rate(), delta(), and increase() functions.
 //
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#rate
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#increase
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#delta
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:55]:
```suggestion
// extrapolatedRate is a helper function that calculates extrapolated rates over
// counters and is used to implement PromQL's
// [`rate()`](https://prometheus.io/docs/prometheus/latest/querying/functions/#rate),
// [`delta()`](https://prometheus.io/docs/prometheus/latest/querying/functions/#increase),
// and [`increase()`](https://prometheus.io/docs/prometheus/latest/querying/functions/#delta) functions.
```

File: stdlib/internal/promql/promql.flux [hunk 15]
```diff
@@ -6,65 +8,94 @@ import "math"
...
+//
+// ## Parameters
+// - tables: Stream of tables.
+// - isCounter: True if the data represents a counter
+// - isRate: True if the data represents a rate
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:60]:
```suggestion
// ## Parameters
// - tables: Input data. Default is piped-forward data (`<-`).
// - isCounter: Data represents a counter.
// - isRate: Data represents a rate.
```

File: stdlib/internal/promql/promql.flux [hunk 16]
```diff
@@ -6,65 +8,94 @@ import "math"
...
         <-tables: stream[{A with _start: time, _stop: time, _time: time, _value: float}],
         ?isCounter: bool,
         ?isRate: bool,
     ) => stream[{B with _value: float}]
 
-// holtWinters() implements functionality equivalent to PromQL's holt_winters()
+// holtWinters implements functionality equivalent to PromQL's holt_winters()
 // function:
 //
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#holt_winters
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:70]:
```suggestion
// holtWinters implements functionality equivalent to
// [PromQL's `holt_winters()` function](https://prometheus.io/docs/prometheus/latest/querying/functions/#holt_winters).
```

File: stdlib/internal/promql/promql.flux [hunk 17]
```diff
@@ -6,65 +8,94 @@ import "math"
...
+//
+// ## Parameters
+// - tables: Stream of tables.
+// - smoothingFactor: Exponential smoothing factor
+// - trendFactor: Trend factor
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:75]:
```suggestion
// - tables: Input data. Default is piped-forward data (`<-`).
// - smoothingFactor: Exponential smoothing factor.
// - trendFactor: Trend factor.
```

File: stdlib/internal/promql/promql.flux [hunk 18]
```diff
@@ -6,65 +8,94 @@ import "math"
...
 builtin holtWinters : (
         <-tables: stream[{A with _time: time, _value: float}],
         ?smoothingFactor: float,
         ?trendFactor: float,
     ) => stream[{B with _value: float}]
 
-// promqlHour() implements functionality equivalent to PromQL's hour() function:
+// promqlHour implements functionality equivalent to PromQL's hour() function:
 //
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#hour
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:84]:
```suggestion
// promqlHour implements functionality equivalent to
// [PromQL's `hour()` function](https://prometheus.io/docs/prometheus/latest/querying/functions/#hour).
```

File: stdlib/internal/promql/promql.flux [hunk 19]
```diff
@@ -6,65 +8,94 @@ import "math"
...
+// ## Parameters
+// - timestamp: Time as a floating point value.
 builtin promqlHour : (timestamp: float) => float
 
-// instantRate() is a helper function that calculates instant rates over
+// instantRate is a helper function that calculates instant rates over
 // counters and is used to implement PromQL's irate() and idelta() functions.
 //
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#irate
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#idelta
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:94]:
```suggestion
// instantRate is a helper function that calculates instant rates over
// counters and is used to implement PromQL's
// [`irate()`](https://prometheus.io/docs/prometheus/latest/querying/functions/#irate) and
// [`idelta()`](https://prometheus.io/docs/prometheus/latest/querying/functions/#idelta) functions.
```

File: stdlib/internal/promql/promql.flux [hunk 20]
```diff
@@ -6,65 +8,94 @@ import "math"
...
+//
+// ## Parameters
+// - tables: Stream of tables.
+// - isRate: True if the data represents a rate
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:98]:
```suggestion
// - tables: Input data. Defaults is piped-forward data (`<-`).
// - isRate: Data represents a rate.
```

File: stdlib/internal/promql/promql.flux [hunk 21]
```diff
@@ -73,6 +104,13 @@ builtin instantRate : (
 // labelReplace implements functionality equivalent to PromQL's label_replace() function:
 //
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#label_replace
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:106]:
```suggestion
// labelReplace implements functionality equivalent to
// [PromQL's `label_replace()` function](https://prometheus.io/docs/prometheus/latest/querying/functions/#label_replace).
```

File: stdlib/internal/promql/promql.flux [hunk 22]
```diff
@@ -73,6 +104,13 @@ builtin instantRate : (
...
+//
+// ## Parameters
+// - tables: Stream of tables.
+// - source: Input label
+// - destination: Output label
+// - regex: Pattern as a regex string
+// - replacement: Replacement value
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:113]:
```suggestion
// - tables: Input data. Default is piped-forward data (`<-`).
// - source: Input label.
// - destination: Output label.
// - regex: Pattern as a regex string.
// - replacement: Replacement value.
```

File: stdlib/internal/promql/promql.flux [hunk 23]
```diff
@@ -86,26 +124,44 @@ builtin labelReplace : (
 //
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#deriv
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#predict_linear
+//
+// ## Parameters
+// - tables: Stream of tables.
+// - predict: True if the output should contain a prediction
+// - fromNow: Time as a floating point value
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:131]:
```suggestion
// - tables: Input data. Default is piped-forward data (`<-`).
// - predict: Output should contain a prediction.
// - fromNow: Time as a floating point value.
```

File: stdlib/internal/promql/promql.flux [hunk 24]
```diff
@@ -86,26 +124,44 @@ builtin labelReplace : (
...
 builtin linearRegression : (
         <-tables: stream[{A with _time: time, _stop: time, _value: float}],
         ?predict: bool,
         ?fromNow: float,
     ) => stream[{B with _value: float}]
 
-// promqlMinute() implements functionality equivalent to PromQL's minute() function:
+// promqlMinute implements functionality equivalent to PromQL's minute() function:
 //
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#minute
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:140]:
```suggestion
// promqlMinute implements functionality equivalent to
// [PromQL's `minute()` function]( https://prometheus.io/docs/prometheus/latest/querying/functions/#minute).
```

File: stdlib/internal/promql/promql.flux [hunk 25]
```diff
@@ -86,26 +124,44 @@ builtin labelReplace : (
...
+//
+// ## Parameters
+// - timestamp: Time as a floating point value.
 builtin promqlMinute : (timestamp: float) => float
 
-// promqlMonth() implements functionality equivalent to PromQL's month() function:
+// promqlMonth implements functionality equivalent to PromQL's month() function:
 //
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#month
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:148]:
```suggestion
// promqlMonth implements functionality equivalent to
// [PromQL's `month()` function](https://prometheus.io/docs/prometheus/latest/querying/functions/#month). 
```

File: stdlib/internal/promql/promql.flux [hunk 26]
```diff
@@ -86,26 +124,44 @@ builtin labelReplace : (
...
+//
+// ## Parameters
+// - timestamp: Time as a floating point value.
 builtin promqlMonth : (timestamp: float) => float
 
-// promHistogramQuantile() implements functionality equivalent to PromQL's
+// promHistogramQuantile implements functionality equivalent to PromQL's
 // histogram_quantile() function:
 //
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#histogram_quantile
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:157]:
```suggestion
// promHistogramQuantile implements functionality equivalent to
// [PromQL's `histogram_quantile()` function](https://prometheus.io/docs/prometheus/latest/querying/functions/#histogram_quantile).
```

File: stdlib/internal/promql/promql.flux [hunk 27]
```diff
@@ -86,26 +124,44 @@ builtin labelReplace : (
...
+//
+// ## Parameters
+// - tables: Stream of tables.
+// - quantile: Quantile to compute in the range 0 to 1.
+// - countColumn: Name of the column containing counts.
+// - upperBoundColumn: Name of the column containing upper bound values.
+// - valueColumn: Name of output column for the value.
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:164]:
```suggestion
// - tables: Input data. Default is piped-forward data (`<-`).
// - quantile: Quantile to compute (`[0.0 - 1.0]`).
// - countColumn: Count column name.
// - upperBoundColumn: Upper bound column name.
// - valueColumn: Output value column name.
```

File: libflux/flux-core/src/bin/fluxdoc.rs [hunk 28]
```diff
@@ -370,24 +370,4 @@ const EXCEPTIONS: &[&str] = &[
...
-    "planner",
-    "testing/chronograf",
-    "testing/expect",
-    "testing/influxql",
-    "testing/kapacitor",
-    "testing/pandas",
-    "testing/prometheus",
-    "testing/promql",
-    "testing/usage",
-    "universe/holt_winters",
```
Comment (sanderson) [inline libflux/flux-core/src/bin/fluxdoc.rs:392]: 💥

File: stdlib/internal/promql/promql.flux [hunk 29]
```diff
@@ -117,24 +173,38 @@ builtin promHistogramQuantile : (
     A: Record,
     B: Record
 
-// resets() implements functionality equivalent to PromQL's resets() function:
+// resets implements functionality equivalent to PromQL's resets() function:
 //
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#resets
+//
+// ## Parameters
+// - tables: Stream of tables.
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:181]:
```suggestion
// resets implements functionality equivalent to
// [PromQL's `resets()` function](https://prometheus.io/docs/prometheus/latest/querying/functions/#resets).
//
// ## Parameters
// - tables: Input data. Defaults is piped-forward data (`<-`).
```

File: stdlib/internal/promql/promql.flux [hunk 30]
```diff
@@ -117,24 +173,38 @@ builtin promHistogramQuantile : (
...
 builtin resets : (<-tables: stream[{A with _value: float}]) => stream[{B with _value: float}]
 
-// timestamp() implements functionality equivalent to PromQL's timestamp() function:
+// timestamp implements functionality equivalent to PromQL's timestamp() function:
 //
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#timestamp
+//
+// ## Parameters
+// - tables: Stream of tables.
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:189]:
```suggestion
// timestamp implements functionality equivalent to 
// [PromQL's `timestamp()` function](https://prometheus.io/docs/prometheus/latest/querying/functions/#timestamp).
//
// ## Parameters
// - tables: Input data. Defaults is piped-forward data (`<-`).
```

File: stdlib/internal/promql/promql.flux [hunk 31]
```diff
@@ -117,24 +173,38 @@ builtin promHistogramQuantile : (
...
 builtin timestamp : (<-tables: stream[{A with _value: float}]) => stream[{A with _value: float}]
 
-// promqlYear() implements functionality equivalent to PromQL's year() function:
+// promqlYear implements functionality equivalent to PromQL's year() function:
 //
 // https://prometheus.io/docs/prometheus/latest/querying/functions/#year
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:194]:
```suggestion
// promqlYear implements functionality equivalent to 
// [PromQL's `year()` function](https://prometheus.io/docs/prometheus/latest/querying/functions/#year).
```

File: stdlib/internal/promql/promql.flux [hunk 32]
```diff
@@ -117,24 +173,38 @@ builtin promHistogramQuantile : (
...
+//
+// ## Parameters
+// - timestamp: Time as a floating point value.
 builtin promqlYear : (timestamp: float) => float
 
-// quantile() accounts checks for quantile values that are out of range, above 1.0 or
+// quantile accounts checks for quantile values that are out of range, above 1.0 or
 // below 0.0, by either returning positive infinity or negative infinity in the `_value`
 // column respectively. q must be a float
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:202]:
```suggestion
// column respectively. `q` must be a float.
```

File: stdlib/internal/promql/promql.flux [hunk 33]
```diff
@@ -117,24 +173,38 @@ builtin promHistogramQuantile : (
...
+//
+// ## Parameters
+// - tables: Stream of tables.
+// - q: Quantile to compute in the range 0 to 1.
+// - method: Name of the quantile method to use.
```
Comment (sanderson) [inline stdlib/internal/promql/promql.flux:207]:
```suggestion
// - tables: Input data. Default is piped-forward data (`<-`).
// - q: Quantile to compute (`[0.0 - 1.0]`).
// - method: Quantile method to use.
```

File: stdlib/internal/testutil/testutil.flux [hunk 34]
```diff
@@ -1,7 +1,24 @@
+// Package testutil provides helper function for writing test cases.
 package testutil
 
 
+// fail causes the current script to fail
```
Comment (sanderson) [inline stdlib/internal/testutil/testutil.flux:5]:
```suggestion
// fail causes the current script to fail.
```

File: stdlib/internal/testutil/testutil.flux [hunk 35]
```diff
@@ -1,7 +1,24 @@
...
 builtin fail : () => bool
+
+// yield is the identity function.
+//
+// ## Parameters
+// - v: Any value.
 builtin yield : (<-v: A) => A
+
+// makeRecord is the identity function but breaks the type connection from input to output.
```
Comment (sanderson) [inline stdlib/internal/testutil/testutil.flux:14]:
```suggestion
// makeRecord is the identity function, but breaks the type connection from input to output.
```

File: stdlib/internal/testutil/testutil.flux [hunk 36]
```diff
@@ -1,7 +1,24 @@
...
+//
+// ## Parameters
+// - o: Record value.
 builtin makeRecord : (o: A) => B where A: Record, B: Record
+
+// makeAny  constructs any value based on a type description as a string
```
Comment (sanderson) [inline stdlib/internal/testutil/testutil.flux:20]:
```suggestion
// makeAny constructs any value based on a type description as a string.
```

File: stdlib/planner/planner.flux [hunk 37]
```diff
@@ -1,5 +1,9 @@
+// Package planner provides an API for interacting with the Flux engine planner
```
Comment (sanderson) [inline stdlib/planner/planner.flux:1]:
```suggestion
// Package planner provides an API for interacting with the Flux engine planner.
```

File: stdlib/testing/expect/expect.flux [hunk 38]
```diff
@@ -13,4 +13,7 @@ package expect
 // exactly as many times as the number given.
 //
 // The key is the name of the planner rule.
+//
+// ## Parameters
+// - rules: Mapping of rules names to expected counts
```
Comment (sanderson) [inline stdlib/testing/expect/expect.flux:18]:
```suggestion
// - rules: Mapping of rules names to expected counts.
```

**Review by sanderson — CHANGES_REQUESTED** [review]
Suggestions for punctuation, formatting, and verbiage changes.

**Review by sanderson — APPROVED** [review]
Looks great!

**Review by rockstar — APPROVED** [review]

**Pull request merged**